### PR TITLE
[DDW-289] Properly handle Ada redemption screen in case there are no wallets

### PR DESCRIPTION
**Pull request by daniloprates**
This PR creates a screen for Ada redemption in case there are no wallets.

## Todo:

- [x] Create the screen based on [this solution](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5a86e931e6774c250a3b0f22).
- [x] Provide English texts [@darko-mijic]
- [x] Provide Japanese translations [**Hiroto**]
- [x] Provide [final design](zpl.io/b6MQmxP) [@a-rukin]

## Screenshots:

image
image
image

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`npm run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`npm run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`npm run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`npm run flow:test`)
- [ ] There are no *lint* errors or warnings (`npm run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `npm run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`npm run storybook`)
- [ ] In case of npm dependency changes both `package-lock.json` and `yarn.lock` files are updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@darko-mijic we will need final copy for this page!

**Comment by nikolaglumac** [comment]
@daniloprates can you please remove the dot in the Button text? Thanks!

PS. please also merge in latest develop branch.

**Comment by nikolaglumac** [comment]
@daniloprates Japanese translations injected:
screen shot 2018-06-14 at 09 45 27

**Comment by nikolaglumac** [comment]
@a-rukin please check if the dark-theme is ok:

screen shot 2018-06-14 at 09 45 38

**Comment by nikolaglumac** [comment]
@daniloprates - bug confirmed. Something is wrong with the routes logic in your branch. If you have no wallets and you go to redemption page and then to add-wallets page it will show up the loading spinner without any content...

**Comment by a-rukin** [comment]
@nikolaglumac dark theme seems to be correct

File: source/renderer/app/stores/WalletStore.js [hunk 1]
```diff
@@ -209,15 +208,6 @@ export default class WalletsStore extends Store {
...
   );
 
-  _showAddWalletPageWhenNoWallets = () => {
-    const { currentRoute } = this.stores.app;
-    const isAdaRedemptionRoute = matchRoute(ROUTES.ADA_REDEMPTION, currentRoute);
-    const isRouteThatNeedsWallets = this.isWalletRoute || isAdaRedemptionRoute;
-    if (isRouteThatNeedsWallets && !this.hasAnyWallets) {
-      this.actions.router.goToRoute.trigger({ route: ROUTES.WALLETS.ADD });
-    }
-  };
```
Comment (DominikGuzei) [inline source/renderer/app/stores/WalletStore.js]: image

**Comment by nikolaglumac** [comment]
Let's merge this one as soon as CI is done.

**Pull request merged**